src/domdiv/cards.py

```diff
@@ -208,11 +209,7 @@ def setImage(self, use_set_icon=False):
                     setImage = Card.sets[self.cardset_tag]["image"]
 
         if setImage is None and self.cardset_tag != "base":
-            print(
-                'warning, no set image for set "{}", card "{}"'.format(
-                    self.cardset, self.name
-                )
-            )
+            logger.warning('no set image for set "{self.cardset}", card "{self.name}"')
```

Comment:
Nice upgrade, no need for f-strings I assume.

Reply:
Actually no, I think this is wrong :). From the docs I initially thought there was no need, but there is. The lib can do `format`-style substitution but then needs the args supplied explicitly, so I went with standard f-strings instead. I'll fix this.